cmd/skywire-cli/commands/visor/info.go

```diff
@@ -14,16 +14,18 @@ import (
 var path string
 var pkg bool
 var web bool
+var webport string
```

Comment:
Camel case.

Reply:
Also, what do you think about just serving an endpoint from the hv UI returning only its public key instead of running this via CLI?

Reply:
it may not be desired in all cases, so it's preferable to have this functionality in the cli